x-pack/plugins/cross_cluster_replication/public/app/services/input_validation.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import Joi from 'joi';
+import { INDEX_ILLEGAL_CHARACTERS_VISIBLE } from 'ui/indices';
+
+export const i18nValidationErrorMessages = {
+  'any.empty': ({ label }) => (
+    i18n.translate('xpack.crossClusterReplication.formInputValidation.errorEmpty', {
+      defaultMessage: `'{label}' is required.`,
```

Comment:
minor: I think the apostrophes are not needed for any of these validation messages, the sentence flows well with just the field name, i.e. `Max read request operation count must be a number.` instead of `'Max read request operation count' must be a number.`

this can be deferred until we do a full copy review 😄

Reply:
Ok, will remove them for now and we'll see what comes up in the copy review